refactor(complete): Initial commit.

kersh1337228 opened the pull request.

kersh1337228 committed:
refactor(complete): Initial commit.

hBuzzy commented on src/widget/widget.cpp:
```diff
@@ -0,0 +1,54 @@
+}
+
+QPoint Widget::KeepCursorInsideWindow(const QPoint& pos) {
+    auto wnd_rect = rect();
```
Давайте все же все переменные в camelCase, без змеиной нотации.

hBuzzy commented on src/web/web.cpp:
```diff
@@ -0,0 +1,29 @@
+    painter.drawLine({boundingRect_.left(), center_y}, cursor_pos);
+    painter.drawLine({boundingRect_.right(), center_y}, cursor_pos);
+
+    painter.drawEllipse(QRect(
+        (boundingRect_.topLeft() + cursor_pos) / 2,
+        (boundingRect_.bottomRight() + cursor_pos) / 2
+    ));
```
В задании, для полного выполнения, разрешены исключительно: `QPainter::DrawLine`, `QPainter::DrawPolyigon` и `QPainter::DrawPolyline`, используйте их.

hBuzzy reviewed (commented):
Принято, но задание не полное. 

Если захотите доделать, то реализуйте дополнительную паутину с помощью разрешенных методов `QPainter`.

Реализуйте круги, например, используя уравнение окружности:
`x = x0 + radius * cos(alpha)`
`y = y0 + radius * sin(alpha)`

Если не хотите использовать его, то просто вместо "кругов" отрисуйте прямоугольники. Это тоже валидный вариант.

kersh1337228 committed:
refactor(web, widget):
[widget] Snake case variable names changed to camel case.
[web] Drawing rectangle instead of ellipse.

kersh1337228 commented:
New commit.

hBuzzy reviewed (commented):
Принято.
